bbn-1/finality-providers/registry/Lombard.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": {
+    "moniker": "Lombard Finance",
```

Comment:
Can you make sure that the moniker does not contains white spaces
https://github.com/babylonlabs-io/networks/blob/main/bbn-1/finality-providers/README.md?plain=1#L228

After updating, you need to resign and update the `Lombard.sig`

Reply:
@fruit37

Reply:
just updated